### PR TITLE
Integration test job

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/giantswarm/issues/9517

### CODE REVIEW

File: integration/test/basic/setup.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+
+#!/bin/bash
```
Comment (rossf7) [inline integration/test/basic/setup.sh:3]: Script has commands that were in the old circle config.

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,48 @@
+module github.com/giantswarm/coredns-app
+
+go 1.13
+
+require (
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // indirect
```
Comment (MarcelMue) [inline go.mod:6]: Not blocking, just wondering where these imports come from. Seems weird to have such an indirect.

**Comment by rossf7** [comment]
I had this PR hanging around. I've updated it after @tomahawk28 migrated to go modules in https://github.com/giantswarm/coredns-app/pull/26

This now uses helmclient v1.0.0 so the go.mod is much leaner.

Could someone take another look?

**Comment by rossf7** [comment]
@MarcelMue Thx! :)

Going with this finally.

**Pull request merged**